### PR TITLE
Various test enhancements

### PR DESCRIPTION
**Pull request by okurz**
* t: Fix warning in full-stack.t
* t: Output isotovideo log files in case of incomplete jobs within the full stack test
* t: Delete superfluous setting delete
* docs: Add hint how to get coverage for individual tests

### CODE REVIEW

File: t/full-stack.t [hunk 1]
```diff
@@ -523,5 +524,8 @@ END {
     kill_driver;
     turn_down_stack;
     session->clean;
+    # prevent double execution
+    return if $? == 0;
```
Comment (kraih) [inline t/full-stack.t]: I still think this is wrong. Somewhere there's an improperly done `fork` causing all sorts of random issues and this masks the most obvious symptom. This needs to be properly investigated.

**Comment by kraih** [comment]
I'll take a look at the fork problem tomorrow.

File: t/full-stack.t (hunk 1 above)
Comment (okurz) [reply t/full-stack.t]: I removed that part now

**Comment by okurz** [comment]
hm, but I doubt the isotovideo output is that long, the debug output with all the web requests are. They are not making the logs easy to read anyway.

**Pull request merged**